pynbody/snapshot/ramses.py

```diff
@@ -794,7 +794,7 @@ def translate_info(sim):
 
     sim.properties['boxsize'] = sim._info['boxlen'] * l_unit
 
-    if sim._info['omega_k'] == sim._info['omega_l'] == sim._info['omega_b'] == 0.0:
```

Comment:
Thanks for the PR - I'm slightly unclear about why checking omega_b is unwanted here?